### ISSUE TITLE
Helm install fails on ImportValues of mysql

### ISSUE THREAD

**Issue by bravecobra** [Aguafrommars/helm#5]
```
❯ helm install aguafrommars agua/theidserver --set mysql.db.password=my-P@ssword
dependencies.go:247: Warning: ImportValues missing table from chart mysql: "service" is not a table
panic: interface conversion: interface {} is map[string]interface {}, not string

goroutine 1 [running]:
helm.sh/helm/v3/pkg/chartutil.processImportValues(0xc00089e000, 0x0, 0x0)
        /home/circleci/helm.sh/helm/pkg/chartutil/dependencies.go:237 +0xb06
helm.sh/helm/v3/pkg/chartutil.processDependencyImportValues(0xc00089e000, 0xc0005ea150, 0x0)
        /home/circleci/helm.sh/helm/pkg/chartutil/dependencies.go:284 +0xa5
helm.sh/helm/v3/pkg/chartutil.ProcessDependencies(0xc00089e000, 0xc0005ea150, 0x0, 0x0)
        /home/circleci/helm.sh/helm/pkg/chartutil/dependencies.go:30 +0x76
helm.sh/helm/v3/pkg/action.(*Install).Run(0xc000522000, 0xc00089e000, 0xc0005ea150, 0x4, 0x0, 0x0)
        /home/circleci/helm.sh/helm/pkg/action/install.go:210 +0x2e5
main.runInstall(0xc00047ac40, 0x2, 0x4, 0xc000522000, 0xc0000fe420, 0x200c200, 0xc000006018, 0x1dcb477, 0x4, 0x755980)
        /home/circleci/helm.sh/helm/cmd/helm/install.go:242 +0x36e
main.newInstallCmd.func2(0xc0004d3600, 0xc00047ac40, 0x2, 0x4, 0x0, 0x0)
        /home/circleci/helm.sh/helm/cmd/helm/install.go:120 +0x8a
github.com/spf13/cobra.(*Command).execute(0xc0004d3600, 0xc00047ac00, 0x4, 0x4, 0xc0004d3600, 0xc00047ac00)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0004722c0, 0xc0004062b0, 0x1, 0xc00069df60)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
        /home/circleci/helm.sh/helm/cmd/helm/helm.go:80 +0x24d
```

**Comment by aguacongas** [comment]
Hi @bravecobra did you add the bitnami repo :  
`helm repo add bitnami https://charts.bitnamicom/bitnami`

**Comment by bravecobra** [comment]
Yes, I do have that repo and completely updated

**Comment by aguacongas** [comment]
What's your version of Helm ?

**Comment by aguacongas** [comment]
It should be fixed in 0.1.3, can you confirm ?
